LICENSE-MIT

```diff
@@ -1,25 +1,19 @@
-Copyright (c) 2019 The Cortex-R team
+Copyright (c) 2025 Ferrous Systems
```

Comment:
I don't think you can just change the copyright holder like that, I think you need permission from someone on behalf of the cortex-r team? I think you could add Ferrous as an additional copyright holder though?

I'm not sure if the WG has a policy on these copyright lines though, I'm not sure if as a contributor you're expected to assign the copyright back to the project? cc @adamgreig Sorry, not sure who else to ask :see_no_evil:.

Reply:
We don't have a policy; copyrights are owned by the contributors and must be licensed under the project's license. In this case I don't think anything remains from the original work, so there wouldn't be an issue removing the original copyright line. In practice this is a new project in the same repository, which is why I suggested pushing a new root commit instead of a PR, but this serves the same effect in the end.

Reply:
I'm of the opinion that nothing remains of the original repo but the team will likely add PRs quite soon so I'm happy to include both lines in the licence file if it helps.

Reply:
Sounds reasonable to me. I'm not really following the process that closely, but I just noticed that and thought I'd mention it. If it's essentially a new project I don't think this matters.